lib/handlers.js

```diff
@@ -96,7 +100,7 @@ module.exports = function (dataset, opts) {
       timeout: opts.timeout || DEFAULT_TIMEOUT,
       guid: opts.guid,
       method: 'GET',
-      path: '/' + dataset + '/' + id.toString()
+      path: pathMod.normalize('/' + opts.pathToPrepend + '/' + dataset + '/' + id.toString())
```

Comment:
Probably should update the tests to verify this path is constructed correctly based on the inclusion or omission of _pathToPrepend_.